### ISSUE TITLE
Not creating fake AP???

### ISSUE THREAD

**Issue by Deepak-Mandal** [wi-fi-analyzer/fluxion#38]
### Which version of fluxion are you using?
2

### What distribution of Linux (including the version) are you running it on?
Kali linux17.3

### What wireless adapter are you using? Exact model and chipset, statements like "internal" and "was working before" are not helpful at all.
Tplink=TL-WN727N(External usb adapter) && chipset=mt7601u Ralink technology.

### Which is the driver for it?
mt7601u

### Does it support injection ( output of aireplay-ng -9)
yes

### General description of your issue along with the steps to reproduce it
At last, Fake AP is not creating and whole process is stuck..
What might be the prblm??suggest me??

**Comment by certifiedrogue** [comment]
Check your tplink WiFi card
I would recommend getting an Alfa AWUSO36NHA card
Works best...
With it you don't need two cards

**Comment by johnbaitsyou** [comment]
can you provide some logs? maybe a dmesg
``
dmesg | tail
``

**Comment by rjshakil1145** [comment]
I'm facing same problem.if anyone have solution,then please help me

**Comment by rjshakil1145** [comment]
![screenshot from 2018-03-19 18-10-21](https://user-images.githubusercontent.com/37431509/37594908-ef7e4820-2ba0-11e8-88c9-095328756c8b.png)

thats the screenshot @johnbaitsyou

**Comment by rjshakil1145** [comment]
My external adapter is same.and thats the screenshot where fluxion is stopped... @johnbaitsyou 
![screenshot from 2018-03-19 18-16-02](https://user-images.githubusercontent.com/37431509/37595131-d4653d86-2ba1-11e8-9aef-14b6c5bab83e.png)

**Comment by rjshakil1145** [comment]
Today i tried capitive portal ap service with my internal wifi card and it worked,but my internal card doesn't support injecting.Then again i tried with my internal card,this time the external card was plugged and i faced this problem again(the capitive portal ap service wasn't starting).i'm little bit upset with this issue.if anyone find solution then please share with me. @johnbaitsyou

**Comment by Shark-A** [comment]
Guys I'm having the same problem on my new HP 8770w laptop im getting really upset Please tell me what to do I tried almost everything

**Comment by rjshakil1145** [comment]
same as me @Shark-A 
r u using the same wifi card?

**Comment by microhasher** [comment]
Hi 
I have the same problem.
Fluxion get stuck at the Fake AP.
I use Fluxion 4 (Online version) with the Alpha AWUS1900 chipset.

**Comment by rjshakil1145** [comment]
yeah,i faced it in fluxion 3.11.about 2 days ago i ran fluxion 4 and still i'm facing this problem.i don't know its a bug or something like driver issue,if anyone have solution then please help us.

**Comment by Shark-A** [comment]
@rjshakil1145 No mine is
 Intel Centrino Ultimate-N 6300 AGN

**Comment by rjshakil1145** [comment]
ooh...!!i'm trying to solve this problem from last few days,but still i dont find any solution.

**Comment by venkyrahul** [comment]
@rjshakil1145 mention ur router model here both of em

**Comment by rjshakil1145** [comment]
my victim is using a tp-link router,i dont know the model.my external card is tp link tl-wn727n which has Ralink Technology, Corp. MT7601U chipset and my internal card is MEDIATEK Corp. MT7630e 802.11bgn Wireless Network Adapter.

**Comment by venkyrahul** [comment]
two external cards make it two ignore the internal card two external cards one to deauth and the other to host. ok

**Comment by rjshakil1145** [comment]
it means i need 2 external cards @venkyrahul

**Comment by rjshakil1145** [comment]
But i see many tutorials in youtube and everyone is using 1 wifi card and successfully collecting the password @venkyrahul

**Comment by rjshakil1145** [comment]
and another think is captive portal ap is starting with all other windows while i'm trying without my external wifi card(it means with only my internal card).but my internal card is unable to catch the password

**Comment by Shark-A** [comment]
@venkyrahul No it does'nt need two wifi cards I used to work with fluxion many times and I succed but on my older laptop.. it woked just fine

**Comment by Shark-A** [comment]
Anyways I'm gonna try adding an external card and let you guys know the result

**Comment by rjshakil1145** [comment]
okk @Shark-A 
i will wait

**Comment by azamet90** [comment]
still no results :D

**Comment by azamet90** [comment]
> okk @Shark-A
> i will wait for know your result.

and still no result :D

**Comment by muslimomar** [comment]
Help plz!
I'm having the same problem! so I installed fluxion yesterday, and it worked perfectly, but today Fake AP is not starting although there are no error logs.

**Comment by sacgeo99** [comment]
> Help plz!
> I'm having the same problem! so I installed fluxion yesterday, and it worked perfectly, but today Fake AP is not starting although there are no error logs.

Did you turn on the monitor mode.
Before turning on monitor mode.
Use: systemctl network.services stop 
This will kill all connections and after this use airmon-ng start wlan"?"
Now run fluxion and everything should work fine
